sql/incoming-schema-changes.sql

```diff
@@ -87,6 +87,7 @@ CREATE TABLE playertimes (
   userid VARCHAR(32) NOT NULL,
   time_in_minutes INT(5) unsigned not null,
   time_note VARCHAR(150),
+  createdate DATE,
```

Comment:
I think `createdate DATE DEFAULT (CURRENT_DATE)` will make sure it's set automatically, so you even don't need to mention it when inserting a row.

(I haven't tested this)

Reply:
[Stack Overflow says](https://stackoverflow.com/questions/20461030/current-date-curdate-not-working-as-default-date-value/51709343#51709343)

> According to this [documentation](https://dev.mysql.com/doc/refman/8.0/en/data-type-defaults.html), starting in MySQL 8.0.13, you will be able to specify:
> 
> CREATE TABLE INVOICE(
>     INVOICEDATE DATE DEFAULT (CURRENT_DATE)
> )

I don't know what version we are using, but it sounds promising.

Reply:
Ok, I changed this.